src/components/map/Map.tsx

```diff
@@ -80,6 +82,9 @@ export const Map = ({
           left: 0
         }}
       >
+        {!!geometryTourData?.length && (
+          <Polyline coordinates={geometryTourData} strokeWidth={2} strokeColor={colors.primary} />
```

Comment:
is there some z index or layering stuff? i have the polyline under the map tiles for some reason.

Reply:
when i change `<UrlTile` and `<Polyline` it works